doc/07-Operations.md

```diff
@@ -0,0 +1,128 @@
+# Operations
+
+Once Icinga DB is installed and configured, it usually runs silently in the background.
+This section is a loose collection of various topics to keep it that way.
```

Comment:
Keep it what way? Running silently in the background?

Reply:
That was loosely what I wanted to say, maybe a bit flippant. A more serious introductory sentence might be good, I guess.

Reply:
Doesn't have to be more serious, I just got a bit confused at the wording :)

Reply:
I have exchanged the introductory sentence to a more descriptive one. Please take another look.

Reply:
Looks good to me!